administrator/components/com_admin/sql/updates/postgresql/5.3.0-2024-12-23.sql

```diff
@@ -2,4 +2,4 @@ ALTER TABLE "#__assets" ALTER COLUMN "name" TYPE varchar(255) NOT NULL;
 ALTER TABLE "#__assets" ALTER COLUMN "title" TYPE varchar(255) NOT NULL;
 ALTER TABLE "#__categories" ALTER COLUMN "extension" TYPE varchar(100) DEFAULT '' NOT NULL;
 ALTER TABLE "#__workflows" ALTER COLUMN "extension" TYPE varchar(255) NOT NULL;
-ALTER TABLE "#__workflow_associations" ALTER COLUMN TYPE"extension" varchar(255) NOT NULL;
\ No newline at end of file
+ALTER TABLE "#__workflow_associations" ALTER COLUMN “extension" TYPE varchar(255) NOT NULL;
```

Comment:
```suggestion
ALTER TABLE "#__workflow_associations" ALTER COLUMN "extension" TYPE varchar(255) NOT NULL;
```
Sorry, my mistake, wrong kind of double quotes used in my previous suggestion.